TWLight/resources/templates/resources/partner_detail.html

```diff
@@ -216,6 +216,26 @@ <h1>{{ object }}</h1>
         </div>
       </div>      
     </li>
+    {% if video_tutorials %}
+      <li class="timeline-inverted">
+        <div class="timeline-badge"><i class="fa fa-file"></i>
+        </div>
+        <div class="timeline-panel">
+          <div class="timeline-body">
+            <h4 class="timeline-title">
+              {% comment %} Translators: If a partner has video tutorials, this text shows as the title for that information field.{% endcomment %}
+              {% trans "Video tutorials" %}
+            </h4>
+            <ul>
+              {% comment %} Translators: If a partner has video tutorials, this text links to the video(s).{% endcomment %}
+              {% for each_tutorial in video_tutorials %}
+                <li><a href={{ each_tutorial | safe }}>{% trans 'Video' %} {{ forloop.counter }}</a></li>
```

Comment:
I _think_ `{{ forloop.counter }}` should be inside the trans tag, to ensure that RTL languages still format the sentence correctly, but I'd appreciate if you could double check in Translatewiki's guidance for developers.